build.sbt

```diff
@@ -2,11 +2,11 @@ organization := "org.http4s"
 name := "jawn-fs2"
 
 scalaVersion := "2.12.8"
```

Comment:
Should we make 2.13 the primary version while we're at it?

Reply:
That's fine with me.